build.gradle

```diff
@@ -1,17 +1,23 @@
 buildscript {
     ext {
-        compose_version = '1.4.3'
+        compose_version = '1.6.0-beta03'
+        material3_version = "1.2.0-beta01"
```

Comment:
The compose version 1.6.0 has been officially released. I think we can start using it.
There is also newer material3_version:1.2.0-rc01

Reply:
Done 👍